src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
-
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        prop={[
```

Comment:
You can call your props anything you like. In this case calling this prop something like `data` or `footerData` would be more descriptive than just calling it `prop`. That could lead to confusion if you have multiple props